### ISSUE TITLE
Simulation of mapmatched route with waypoints jumps to 0,0

### ISSUE THREAD

**Issue by wbearand** [mapbox/mapbox-navigation-android#1678]
Android API: 28
**Mapbox Navigation SDK version: 0.27.0-SNAPSHOT

### Steps to trigger behavior
 1. Start mapmatched route with waypoints in simulation mode
 2. After going straight, at first turn the map suddenly pans to water for a second (which turned out to be 0,0) and then jumps back and resumes the simulation

```
2019-01-15 22:17:07.914 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,782768,-122,486015 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:08.957 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,782864,-122,486023 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:10.027 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783009,-122,486038 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:11.074 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783108,-122,486038 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:12.147 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783203,-122,486046 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:13.211 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783348,-122,486061 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:14.250 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783447,-122,486069 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:15.316 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783409,-122,486061 acc=3 et=?!? vel=12.5 bear=356.88303]
2019-01-15 22:17:16.347 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 0,000000,0,000000 acc=3 t=?!? et=?!? vel=12.5 bear=0.0]
2019-01-15 22:17:17.410 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783833,-122,486092 acc=3 et=?!? vel=12.5 bear=329.52164]
2019-01-15 22:17:18.443 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783833,-122,486092 acc=3 et=?!? vel=12.5 bear=329.52164]
2019-01-15 22:17:19.491 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783833,-122,486092 acc=3 et=?!? vel=12.5 bear=329.52164]
2019-01-15 22:17:20.529 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783840,-122,486099 acc=3 et=?!? vel=12.5 bear=86.666595]
2019-01-15 22:17:21.589 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783848,-122,485954 acc=3 et=?!? vel=12.5 bear=86.666595]
2019-01-15 22:17:22.624 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783859,-122,485657 acc=3 et=?!? vel=12.5 bear=86.666595]
2019-01-15 22:17:23.655 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783867,-122,485504 acc=3 et=?!? vel=12.5 bear=86.666595]
2019-01-15 22:17:24.686 21522-21522/com.......................................... D/WaypointNavigationActivity: onProgressChange - location: Location[ReplayRouteLocation 37,783875,-122,485352 acc=3 et=?!? vel=12.5 bear=86.666595]
```

### Expected behavior
Map in navigation simulation should not jump to 0,0 for a second.

**Comment by danesfeder** [comment]
@wbearand can you retest this with `0.30.0`? Thanks!

**Comment by danesfeder** [comment]
Hey @wbearand 👋 this should be resolved in `0.30.0` if you're able to retest and confirm, that would be great.  If you still see issue, please open a new ticket.  Going to close here as stale.
